### DISCUSSION TITLE
SCAR INSTANT Workshop Poster

### DISCUSSION THREAD

**Discussion by mdtanker** [mdtanker/antarctic_plots#81] [Show and tell]
Here is a poster presented at the 2022 SCAR INSTANT workshop in Fort Collins, Colorado: The Future of Geodetic-Geophysical Observational Networks in Antarctica. The poster was advertising this package, specifically version 0.0.4.2. 

The code for all the figures in the poster are available in [this notebook](https://github.com/mdtanker/SCAR_INSTANT_2022/blob/main/figures/Poster_plots.ipynb).
Poster link: [figshare](https://figshare.com/articles/poster/Poster_Antarctic-Plots_A_Python_package_to_help_download_visualize_and_present_Antarctic_datasets/21183931)
Github repo: [here](https://github.com/mdtanker/SCAR_INSTANT_2022)

[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/mdtanker/SCAR_INSTANT_2022/v1.0?labpath=figures/Poster_plots.ipynb)

![Tankersley_INSTANT_2022](https://user-images.githubusercontent.com/81199856/193417562-9d5b846e-0a7b-48c8-bd01-d1b0b6dd3547.png)

**Comment by Belial2010** [comment]
来信已收到，我会尽快回复您信息。

**Comment by Carolina-98** [comment]
Nice poster!
